### ISSUE TITLE
[css-images] Clarification about `Nx` as <resolution> value in image-set() is needed

### ISSUE THREAD

**Issue by lahmatiy** [w3c/csswg-drafts#461]
Current syntax for `image-set()` is:

```
image-set() = image-set( <image-set-option># )
<image-set-option> = [ <image> | <string> ] <resolution>
```

Where [`<resolution>`](https://drafts.csswg.org/css-values-3/#resolution-value) refers to `CSS Values and Units Module Level 3` where it described as dimension with `dpi`, `dpcm` or `dppx` unit. However in example below we can see (that even works in some browsers):

```
background-image: image-set( "foo.png" 1x,
                             "foo-2x.png" 2x,
                             "foo-print.png" 600dpi );
```

I can't find reference where it says that dimension with `x` unit can be used as `<resolution>`. Looks like this is `image-set()` specific or I'm wrong. In any case, clarification is needed here.

**Comment by fantasai** [comment]
Lea, this isn't editorial, it's substantive, and needs WG resolution.

**Comment by LeaVerou** [comment]
@fantasai No, it's not. The syntax doesn't change, it's only a bugfix on the grammar.

**Comment by LeaVerou** [comment]
Resolved via ccde9f0ae68fa9dd7b6250ef71bf78cce4be81ef
